docs/en/sql-reference/statements/grant.md

```diff
@@ -13,24 +13,26 @@ To revoke privileges, use the [REVOKE](../../sql-reference/statements/revoke.md)
 ## Granting Privilege Syntax {#grant-privigele-syntax}
 
 ``` sql
-GRANT [ON CLUSTER cluster_name] privilege[(column_name [,...])] [,...] ON {db.table|db.*|*.*|table|*} TO {user | role | CURRENT_USER} [,...] [WITH GRANT OPTION]
+[REPLACE] GRANT [ON CLUSTER cluster_name] privilege[(column_name [,...])] [,...] ON {db.table|db.*|*.*|table|*} TO {user | role | CURRENT_USER} [,...] [WITH GRANT OPTION]
 ```
 
 -   `privilege` — Type of privilege.
 -   `role` — ClickHouse user role.
 -   `user` — ClickHouse user account.
 
+The `REPLACE` clause replace old privileges by new privileges for the `user` or `role`
```

Comment:
Probably we should add a note here that if `GRANT` is used without `REPLACE` it adds priviliges to the currently existing privileges. Yes, it's kind of obvious but yet it might be useful.

Reply:
good advice